packages/legend-extension-format-graphql/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# @finos/legend-extension-format-graphql
```

Comment:
let's delete this file, this package is never released, it should not have a changelog file

Reply:
done